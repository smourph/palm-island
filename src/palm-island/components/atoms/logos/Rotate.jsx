import React from 'react';
import rotate from '@internal/resources/images/logo-rotate.png';
import './Logo.scss';

export const Rotate = () => (
  <span className="logo logo-action logo-rotate">
      <img src={rotate} alt="rotate"/>
  </span>
);

export default Rotate;
