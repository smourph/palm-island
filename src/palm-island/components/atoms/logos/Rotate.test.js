import React from 'react';
import {render, screen} from '@testing-library/react';
import Rotate from './Rotate';

it('should render snapshot', () => {
  const {asFragment} = render(<Rotate/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render rotate', () => {
  render(<Rotate/>);

  const icon = screen.getByAltText('rotate');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-rotate.png');
});
