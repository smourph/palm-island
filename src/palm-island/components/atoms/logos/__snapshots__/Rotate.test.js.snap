// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-action logo-rotate"
>
  <img
    alt="rotate"
    src="logo-rotate.png"
  />
</span>
`;
